backends/backends.c

```diff
@@ -697,6 +697,7 @@ void *backends_main(void *ptr) {
     // prepare the backend main loop
 
     info("BACKEND: configured ('%s' on '%s' sending '%s' data, every %d seconds, as host '%s', with prefix '%s')", type, destination, source, global_backend_update_every, hostname, global_backend_prefix);
+    send_statistics("BACKENDS_START", "OK", type);
```

Comment:
all the `info` and` and `debug` messages are in the singular ( `BACKEND:` ) why are you using plural here?

Reply:
I used it because the name of the subsystem is "backends". But, as far as a specific backend is starting here and taking into account what you said, I've changed the name.